test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,19 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
+    feed('(())')
+    wait()
```

Comment:
wait shouldn't be used prior to `expect()` (I'm also suspicious of the one before, `screen:sleep()` is better if we really need to simulate "the user stopped typing for some while", but I wouldn't expect that here, `input()` should be sync in its operations)